lib/govuk_component/warning_text_helpers.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module GovukComponent
+  module WarningTextHelpers
```

Comment:
**not a blocker** - just in terms of naming - have you considered a "higher" level name that could be used to add other similar helpers - such as a for notice or alert type messages

Reply:
I have had a similar discussion with design (Dan) as CCCD currently uses this component for warning / note / information.
We have decided to keep it consistent with the Govuk Design System and have further discussions later for a "higher" level name.